components/admin_console/billing/billing_subscriptions/contact_sales_card.tsx

```diff
@@ -159,7 +159,7 @@ const ContactSalesCard = (props: Props) => {
                 {(!isFreeTrial && subscriptionPlan !== CloudProducts.ENTERPRISE && subscriptionPlan !== CloudProducts.LEGACY) &&
                     <button
                         type='button'
-                        onClick={onUpgradeMattermostCloud}
+                        onClick={() => onUpgradeMattermostCloud('admin_console_billing_contact_sales_card_upgrade_now_button')}
```

Comment:
This Contact Sales card I believe it changes to Upgrade (with no reference to sales) when not in Enterprise. Removed the reference to Contact Sales as is misleading

```suggestion
                        onClick={() => onUpgradeMattermostCloud('admin_console_subscription_card_upgrade_now_button')}
```